### PR TITLE
Adding RedHat Connect Summit blog and supporting images

### PR DESCRIPTION
**Pull request by gljrobinson**

### CODE REVIEW

**Review by Yas-Franco — DISMISSED** [review]
LGTM

File: posts/2021-11-30-RedHat-Connect-London.adoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+layout: post
+title: "A day out in London at the RedHat Connect Summit"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/yasmin-aumeeruddy
+author_github: https://github.com/yasmin-aumeeruddy
+seo-title: A day out in London at the RedHat Connect Summit
+seo-description: 
+blog_description: 
+open-graph-image: https://openliberty.io/img/AndyMcCright-1.png
```
Comment (dmuelle) [inline posts/2021-11-30-RedHat-Connect-London.adoc]: need to change the open graph image to be one of the images from the post, in this format:

https://openliberty.io/img/blog/image-filename.png

File: posts/2021-11-30-RedHat-Connect-London.adoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+---
+= A day out in London at the RedHat Connect Summit
+Yasmin Aumeeruddy <https://github.com/yasmin-aumeeruddy>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+link:https://twitter.com/gracejansen27[Grace Jansen] and link:https://github.com/yasmin-aumeeruddy[Yasmin Aumeeruddy] from the Open Liberty team joined our cousins at RedHat to participate in the link:https://www.redhat.com/en/summit/connect/emea/london/developers[RedHat Connect Summit in London]. This event was a single-day event, offering a variety of developer-focused sessions, many of which utilised or focused on OSS projects and initiatives. The topics of the sessions ranged from those focused on cloud-native development, to DevOps, to enterprise development, to more general language and framework sessions. As well as the conference sessions there were also lightning talks occurring in the expo area and live coding and arcade games as well! These one-day events are run in cities all over the globe. You can check out if there's an event occurring near you at their link:https://www.redhat.com/en/summit/connect#locations--dates[website].
```
Comment (dmuelle) [inline posts/2021-11-30-RedHat-Connect-London.adoc]: suggested edit:
This event was a single-day event, offering a variety of developer-focused sessions, many of which utilised or focused on OSS projects and initiatives. The topics of the sessions ranged from those focused on cloud-native development, to DevOps, to enterprise development, to more general language and framework sessions.

--->

This single-day event offered a variety of developer-focused sessions, many of which drew from open source software (OSS) projects and initiatives. The topics included cloud-native development, DevOps and  enterprise development, as well as more general language and framework sessions.


As well as the conference sessions there were also lightning talks occurring in the expo area and live coding and arcade games as well!
--->
The summit also featured lightning talks in the expo area, live coding, and even arcade games!

File: posts/2021-11-30-RedHat-Connect-London.adoc [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Although audiences have engaged well with virtual conferences over the past 18 months, it was great to attend an in-person event. For many, it was the first in a long time but it was a fantastic way to re-engage with the community and enable attendees to network with OSS and enterprise software developers from many different organisations.
+
+
+== Open Liberty sessions:
+Our Open Liberty advocates had two sessions accepted at this event - 'Reacting to our data driven world' and 'Replicating production in your tests with the magic of containers'. These talks were limited to 35 minutes in length but were full of great insights and demonstrations. We had a great turnout at both of these sessions, with interesting questions raised by the audience and many even staying to view our demos during their lunch break!
```
Comment (dmuelle) [inline posts/2021-11-30-RedHat-Connect-London.adoc]: suggested edit:

Our Open Liberty advocates had two sessions accepted at this event - 'Reacting to our data driven world' and 'Replicating production in your tests with the magic of containers'. These talks were limited to 35 minutes in length but were full of great insights and demonstrations.
--->
Our Open Liberty advocates led two sessions at this event: _Reacting to our data driven world_ and _Replicating production in your tests with the magic of containers_. These talks were only 35 minutes long but were full of great insights and demonstrations.

File: posts/2021-11-30-RedHat-Connect-London.adoc [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+== Open Liberty sessions:
+Our Open Liberty advocates had two sessions accepted at this event - 'Reacting to our data driven world' and 'Replicating production in your tests with the magic of containers'. These talks were limited to 35 minutes in length but were full of great insights and demonstrations. We had a great turnout at both of these sessions, with interesting questions raised by the audience and many even staying to view our demos during their lunch break! 
+
+image::/img/blog/RedHatConnectSummitLondon-1.png[Photograph of Grace presenting first Open Liberty session,width=70%,align="center"]
+
+The first session focused on creating event-driven reactive microservices through the use of OSS tools like Kafka, MicroProfile and Open Liberty. The audience were provided with an understanding of what the app architecture could offer to their own applications and then they were guided through how they could start implementing it themselves. If you're interested in this yourself, you can try out our Open Liberty guide that walks you through this: link:https://openliberty.io/guides/microprofile-reactive-messaging.html['Creating reactive Java microservices'].
```
Comment (dmuelle) [inline posts/2021-11-30-RedHat-Connect-London.adoc]: If you're interested in this yourself, you can try out our Open Liberty guide that walks you through this: link:https://openliberty.io/guides/microprofile-reactive-messaging.html['Creating reactive Java microservices']
--->
If you're interested, you can try out our link:https://openliberty.io/guides/microprofile-reactive-messaging.html['Creating reactive Java microservices'] guide, which walks you through this process.

File: posts/2021-11-30-RedHat-Connect-London.adoc [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+image::/img/blog/RedHatConnectSummitLondon-2.png[Screenshot of presentation,width=70%,align="center"]
+
+The other session focused on creating true-to-production tests for microservices with OSS tools like Microshed testing, test containers and Open Liberty. It's evident that containers have revolutionised how we design and build our applications but with this move to containers, the parity between our development, testing and production environments has drifted even further apart. The aim of this session was to show developers how to effectively test our cloud-native applications. We also gave a demo to show how simple it is to create and run a MicroShed test in Maven using our link:https://openliberty.io/guides/microshed-testing.html['Testing a MicroProfile or Jakarta EE application'] Open Liberty guide.
```
Comment (dmuelle) [inline posts/2021-11-30-RedHat-Connect-London.adoc]: Microshed testing, test containers and Open Liberty
--->
https://microshed.org/microshed-testing/[Microshed testing], https://www.testcontainers.org/[Testcontainers], and Open Liberty

It's evident that containers have revolutionised how we design and build our applications but with this move to containers, the parity between our development, testing and production environments has drifted even further apart.
--->
Although moving to containers has revolutionised how we design and build our applications,  the parity between our development, testing and production environments has also drifted even further apart.

File: posts/2021-11-30-RedHat-Connect-London.adoc [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+If you're interested in learning more about these innovative technologies and the various types of testing you could implement, then check out our series on IBM Developer: link:https://developer.ibm.com/series/testing-cloud-native-java-applications/['Testing your cloud-native, Java applications'].
+
+
+== Other great sessions:
+There were also loads of other amazing sessions from industry experts and deep technical innovators! Check out the agenda here to see what other sessions were on:
```
Comment (dmuelle) [inline posts/2021-11-30-RedHat-Connect-London.adoc]: Check out https://www.redhat.com/en/summit/connect/emea/london/developers#agenda[the agenda] to see what other sessions were presented.

**Review by dmuelle — CHANGES_REQUESTED** [review]
a few minor suggestions

**Review by dmuelle — APPROVED** [review]

**Pull request merged**